.github/workflows/build.yml

```diff
@@ -24,4 +24,18 @@ jobs:
     - name: Build
       shell: bash
       run: sudo bash build.sh init && bash build.sh release --make -j4
+
+  build-on-mac:
+    # The CMake configure and build commands are platform agnostic and should work equally well on Windows or Mac.
+    # You can convert this to a matrix build if you need cross-platform coverage.
+    # See: https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/managing-complex-workflows#using-a-build-matrix
+    runs-on: mac-latest
```

Comment:
It seems a wrong name.
Please refer to https://docs.github.com/en/actions/using-github-hosted-runners/using-larger-runners